opendcs-rest-api/src/test/java/org/opendcs/odcsapi/fixtures/SingleSignOnWrapper.java

```diff
@@ -23,15 +23,18 @@
 import org.apache.catalina.connector.Request;
 import org.apache.catalina.connector.Response;
 
-public class SingleSignOnWrapper extends SingleSignOn{
-    public void wrappedRegister(String ssoId, Principal principal, String authType,
-    String username, String password) {
-        this.register(ssoId, principal, authType, username, password);
-    }
+public class SingleSignOnWrapper extends SingleSignOn
```

Comment:
shouldn't we be able to just turn the sso valve on in the tomcat setup?

Reply:
I copied this code from CDA. Isn't this bypassing the need to add a separate service that performs the auth?

Reply:
Well, I don't remember that at all, but it definitely makes sense.